sml_builder/help_centre.py

```diff
@@ -47,7 +51,7 @@ def guidances(category, sub_category=None):
         category_label, sub_category_label, sub_category = _get_category_labels(
             category, sub_category
         )
-    except Exception as e:
+    except Exception as e:  # pylint: disable=broad-except
```

Comment:
Pylint flags this line because we aren't exactly specifying what type of error we are catching, it's a broad exception. According to the flask convention, we are doing things correctly by using abort(404) that's provided by flask.